pkg/entity/project/cmd.go

```diff
@@ -33,3 +33,22 @@ func CreateCmd() *cobra.Command {
 	flags.String(descriptionFlag, "", "description of the project")
 	return project
 }
+
+func ManageCmd() *cobra.Command {
+	project := &cobra.Command{
+		Use:   "project",
+		Short: "Manage a project",
```

Comment:
Should we add a more elaborate description?

Reply:
yes, forgot about this